common/matrix/batch_dense_kernels.hpp.inc

```diff
@@ -147,11 +122,38 @@ __device__ __forceinline__ void compute_norm2(
     const gko::batch_dense::BatchEntry<const ValueType> &x,
     const gko::batch_dense::BatchEntry<remove_complex<ValueType>> &result)
 {
-    for (int col = 0; col < x.num_rhs; col++) {
-        one_norm2(x, col, result.values);
+    using real_type = typename gko::remove_complex<ValueType>;
+    constexpr auto tile_size = config::warp_size;
+
+    auto thread_block = group::this_thread_block();
+
+    auto subwarp_grp = group::tiled_partition<tile_size>(thread_block);
+
+    const auto subwarp_grp_id = static_cast<int>(threadIdx.x / tile_size);
+
```

Comment:
Nit: you don't need to leave that many empty lines.
```suggestion
    auto subwarp_grp = group::tiled_partition<tile_size>(thread_block);
    const auto subwarp_grp_id = static_cast<int>(threadIdx.x / tile_size);
```

Reply:
Removed all unnecessary lines.